.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing to Linux Guest Agent
```

Comment:
This is **Azure Functions Python Library**, not Linux Guest Agent

Reply:
Good catch. I picked up from the template. I'll update this.